resources/views/components/base/form-actions.blade.php

```diff
@@ -10,7 +10,7 @@
         {{ $submitLabel }}
     </x-base.button.submit>
 
-    @if($hasRequiredFields)
+    @if ($hasRequiredFields)
```

Comment:
Interesting, blade templates are mixed in the docs with spaces. i'm not used to those, is this a preset in a formatter?

Reply:
Just me, force of habit 😅
Happy to revert to `@if(...condition)` if preferred. The docs do seem to favor a space between for `@if`, but not for other directives. I have to confess I don't understand the nuance.

https://laravel.com/docs/11.x/blade#if-statements

Reply:
Same, only now did I notice that the docs are not consistent in that regard.
Nevermind, let's keep it like this and address this with a linter for blade files as soon as a useful/working tool and preset come along.